### PR TITLE
chore: remove opa and envoy(MAPCO-3324)

### PR DESCRIPTION
**Pull request by almog8k**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: helm/templates/deployment.yaml [hunk 1]
```diff
@@ -73,50 +72,6 @@ spec:
           resources:
             {{- toYaml .Values.resources.value | nindent 12 }}
           {{- end }}
-        {{- if .Values.rasterCommon.authentication.opa.enabled }}
```
Comment (asafMasa) [inline helm/templates/deployment.yaml:76]: run helm dry run
this if wasn't deleted till the -end closure

File: helm/values.yaml [hunk 2]
```diff
@@ -46,36 +42,6 @@ image:
...
-      limits:
-        cpu: 100m
-        memory: 128Mi
-      requests:
-        cpu: 100m
-        memory: 128Mi
-
 env:
   port: 8080
   internalPort: 80
```
Comment (asafMasa) [inline helm/values.yaml:47]: what is the use of "internalPort"?

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: helm/values.yaml (hunk 2 above)
Comment (almog8k) [reply helm/values.yaml:47]: currently there is no use for this parameter, i think someone wanted to implement support of no-auth route but right now it is not implemented

**Pull request closed without merging**